operator/config/grafana/kustomization.yaml

```diff
@@ -4,6 +4,6 @@ kind: Component
 resources:
 - controller-resources-metrics-configmap.yaml
 - controller-runtime-metrics-configmap.yaml
-- kcp-dashboard-configmap.yaml
+- manager-configmap.yaml
 generatorOptions:
-  disableNameSuffixHash: true
+  disableNameSuffixHash: true
```

Comment:
what does this do?

Reply:
as the name described, it will stop add an random name suffix, without this option, the name will become something like `name-chmdhgkb7t`